test/tstDetailsTreeConstruction.cpp

```diff
@@ -195,8 +195,14 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(example_tree_construction, DeviceType,
   };
 
   typename DeviceType::execution_space space{};
+
+  Kokkos::View<Box *, DeviceType> primitives("primitives", n);
+  Kokkos::View<unsigned int *, DeviceType> permutation_indices("indices", n);
```

Comment:
Time to develop a new habit:
```suggestion
  Kokkos::View<Box *, DeviceType> primitives("Testing::primitives", n);
  Kokkos::View<unsigned int *, DeviceType> permutation_indices("Testing::indices", n);
```

Reply:
I edited all views in that unit test.  Not fixing everywhere.